src/components/entropy/index.js

```diff
@@ -147,9 +147,8 @@ class Entropy extends React.Component {
                 zoomMin: this.state.chart.zoomBounds[0],
                 zoomMax: this.state.chart.zoomBounds[1],
               })
-            } else {
-              this.props.dispatch(changeEntropyCdsSelection(nucleotide_gene));
             }
+            this.props.dispatch(changeEntropyCdsSelection(nucleotide_gene));
```

Comment:
With this change (and the earlier change), the following just above this line becomes unnecessary and can be removed:
```js
            if (viewingGenome) {
              this.state.chart.update({
                zoomMin: this.state.chart.zoomBounds[0],
                zoomMax: this.state.chart.zoomBounds[1],
              })
            }
```

This is because we end up running `update({selectedPositions: []})` which runs
```js
    if (selectedCds || selectedPositions !== undefined) {
      this._setZoomCoordinates(zoomMin, zoomMax, !!selectedCds);
    }
```

so we'll always reset the coordinates to show the entire nuc span.

Reply:
It's still used for when no CDS or position is selected, when the min/max sliders are adjusted. I think the update can happen elsewhere though. Started with 1f40cd0d5caaecc1b7ae47e7bb2e707b2ed92452 but haven't gotten it fully working.